handler/oauth2/strategy_jwt.go

```diff
@@ -105,7 +142,25 @@ func (h *RS256JWTStrategy) generate(tokenType fosite.TokenType, requester fosite
 		return "", "", errors.New("GetTokenClaims() must not be nil")
 	} else {
 		claims := jwtSession.GetJWTClaims()
-		claims.ExpiresAt = jwtSession.GetExpiresAt(tokenType)
+
+		if claims.ExpiresAt.IsZero() {
+			claims.ExpiresAt = jwtSession.GetExpiresAt(tokenType)
+		}
+
+		if claims.IssuedAt.IsZero() {
+			claims.IssuedAt = time.Now()
+		}
+
+		if claims.Issuer == "" {
+			claims.Issuer = h.Issuer
+		}
```

Comment:
There seemed to be a lot of cases where these things would be left empty, making it difficult to use JWTs interchangeably in tests.

Noticed the OpenID strategy had this Issuer field, which I added here just because it's handy.